web/app/view/StateController.js

```diff
@@ -184,5 +184,30 @@ Ext.define('Traccar.view.StateController', {
             this.position = null;
             Ext.getStore('Attributes').removeAll();
         }
+    },
+
+    onCellClick: function (grid, td, cellIndex, record, tr, rowIndex, e) {
+        var positionId = this.position.getId();
+        if (record.get('name') === Strings.positionAddress && !record.get('value') &&
+                e.target.tagName.toLowerCase() === 'a') {
```

Comment:
We need to determine if user clicked on link (not other place of cell) to leave behavior intuitive.

Reply:
Can we assign some class to the element instead?